src/app/core/components/event-card/event-card.component.html

```diff
@@ -0,0 +1,81 @@
+<ng-container [ngSwitch]="type">
+  <ng-container *ngSwitchCase="'list-small'">
+    <ng-container *ngTemplateOutlet="list_small"></ng-container>
+  </ng-container>
+  <ng-container *ngSwitchCase="'list-regular'">
+    <ng-container *ngTemplateOutlet="list_regular"></ng-container>
+  </ng-container>
+  <ng-container *ngSwitchCase="'full-card'">
+    <ng-container *ngTemplateOutlet="full_card"></ng-container>
+  </ng-container>
+  <ng-container *ngSwitchDefault>
+    <ng-container *ngTemplateOutlet="pocket"></ng-container>
+  </ng-container>
+</ng-container>
+
+<ng-template #pocket>
```

Comment:
Can we refactor #pocket and #list_small to new components (more on this in other comments)